django_afip/models.py

```diff
@@ -122,7 +126,11 @@ def first_currency() -> int | None:
 def _get_storage_from_settings(setting_name: str) -> Storage:
     path = getattr(settings, setting_name, None)
     if not path:
-        return import_string(settings.DEFAULT_FILE_STORAGE)()
+        if DJANGO_VERSION >= (4,2):
+            from django.core.files.storage import default_storage
+            return default_storage
+        else:
+            return import_string(settings.DEFAULT_FILE_STORAGE)()
```

Comment:
La versión más vieja soportada es 4.2, así que no necesitamos el `else`:

```suggestion
        from django.core.files.storage import default_storage
        return default_storage
```

El import puede ir al principio del archivo.